driver/driver_test.go

```diff
@@ -248,11 +250,72 @@ type fakeStorageActionsDriver struct {
 }
 
 func (f *fakeStorageActionsDriver) Attach(ctx context.Context, volumeID string, dropletID int) (*godo.Action, *godo.Response, error) {
-	return nil, godoResponse(), nil
+	resp := godoResponse()
+
+	if _, ok := f.volumes[volumeID]; !ok {
+		resp.Response = &http.Response{
+			StatusCode: http.StatusNotFound,
+		}
+		return nil, resp, errors.New("volume was not found")
+	}
+
+	droplet, ok := f.droplets[dropletID]
+	if !ok {
+		resp.Response = &http.Response{
+			StatusCode: http.StatusNotFound,
+		}
+		return nil, resp, errors.New("droplet was not found")
+	}
+
+	if len(droplet.VolumeIDs) == maxVolumesPerNode {
```

Comment:
Should we use `>= maxVolumesPerNode` here just to be safe?

Reply:
Yep, good call -- done.